packages/shared/src/components/cards/PostMetadata.tsx

```diff
@@ -35,10 +33,16 @@ export default function PostMetadata({
         className,
       )}
     >
-      {!!post.createdAt && <time dateTime={post.createdAt}>{date}</time>}
-      {!!post.createdAt && !!post.readTime && <Separator />}
-      {!!post.readTime && (
-        <span data-testid="readTime">{post.readTime}m read time</span>
+      {!!createdAt && <span>{date}</span>}
```

Comment:
Why don't we use the `time` element here anymore? Semantically it's the correct thing to do